apidoc/layouts/api.php

```diff
@@ -59,7 +59,7 @@
     <div class="col-sm-8 col-md-9 col-lg-9" role="main">
         <!-- YII_VERSION_SELECTOR -->
         <div class="content">
-        <?= $content ?>
+        <?= $content = str_replace('<a href="index.html">', '<a href="/doc/api/2.0">', $content) ?>
```

Comment:
This will hardcode URL and make it impossible to test this website in subdirectory. It also may result in weird side effects, since you're blindly replacing everything - it may affect code samples or other stuff.

It makes more sense to create redirection from `https://www.yiiframework.com/doc/api/2.0/index.html` to `https://www.yiiframework.com/doc/api/2.0`. Or fix this URL in apidoc.